### ISSUE TITLE
Scale footer positioning issue

### ISSUE THREAD

**Issue by pomoalex** [telekom/scale#770]
After updating to the latest version of Scale, I noticed the footer was no longer positioned at the bottom of the page(as it was before updating from 3.0.0-beta.38), but instead it appears directly after the content.

This is the general structure I have:
```
<scale-app-shell>
     <scale-app-header slot="header"></...>
     <div clas="...">Content</div>
     <scale-app-footer slot="footer"></...>
</scale-app-shell>
```

And this is the result:

![image](https://user-images.githubusercontent.com/22682250/148384123-35d6c1b9-6957-4db8-bc59-a16a29bdf84f.png)

**Comment by nowseemee** [comment]
Hi @pomoalex, thanks for opening this issue - we will take a look at what happened and come back to you with the solution.

**Comment by nowseemee** [comment]
This should be fixed in the latest version `3.0.0-beta.41` - thanks to @maomaoZH for providing the patch and thanks @pomoalex for opening this issue!
